addons/mission/functions/fnc_bomber.sqf

```diff
@@ -28,9 +28,10 @@ if (!is3DENPreview && {!isServer}) exitWith {};
 _unit addVest "Umi_Bomb_Vest_Camo";
 
 [QACEGVAR(common,setSpeaker), [_unit, "ACE_NoVoice"]] call CBA_fnc_globalEvent;
+[QGVAR(setCombatBehaviour), [_unit, "CARELESS"]] call CBA_fnc_globalEvent;
```

Comment:
Should be `targetEvent` at `_unit`. Local Arguments, Global Effects.

Reply:
I did it globally because this could run before HC switch

Reply:
Does locality switch change combat behaviour? I couldn't find any reference about that.

Reply:
I'd assume so based on how everything else seems to work. It only happens once and probably best to cover the bases.